roles/add_day2_node/defaults/main.yml

```diff
@@ -5,6 +5,7 @@ ASSISTED_INSTALLER_BASE_URL: "{{ secure | ternary('https', 'http') }}://{{ ASSIS
 URL_ASSISTED_INSTALLER_CLUSTER: "{{ ASSISTED_INSTALLER_BASE_URL }}/clusters/{{ add_host_cluster_id }}"
 
 day2_states:
+  -  "insufficient"
```

Comment:
quotes aren't needed here but if you want to use them let's wrap the rest of the strings in quotes as well.